posix.ipkg

```diff
@@ -3,6 +3,14 @@ package posix
 sourcedir = src
 modules = System.Posix.Directory
         , System.Posix.Time
+	, System.Posix.Signal
+	, System.Posix.Syslog
+	, System.Posix.Test.Signal
+	, System.Posix.Test.Syslog
```

Comment:
It looks like you have hard tabs here. That might cause problems, but I don’t know how the ipkg parser reacts to them.